lib/before_bedtime.ex

```diff
@@ -0,0 +1,20 @@
+defmodule BabySitter.BeforeBedTime do
+  @before_bed_rate 12
+
+  def calculate_before_bed_pay(start_time, end_time, bed_time) do
```

Comment:
`Bedtime`, `BeforeBedtime`, and `AfterMidnight` could all have the same interface, if you made some mild adjustments. I’d look at a [behavior](https://elixir-lang.org/getting-started/typespecs-and-behaviours.html#behaviours) with a `calculate_pay` method.